its/ruling/src/test/resources/sonar-server/squid-S3655.json

```diff
@@ -12,12 +12,17 @@
 ],
 'org.sonarsource.sonarqube:sonar-server:src/main/java/org/sonar/server/es/EsClientProvider.java':[
 52,
+64,
```

Comment:
All the issues are legit to me. Previously only the first access on an optional was reported, now all the ones which are following in the code. I do NOT consider them as noise.

Reply:
That seems arguable but ok.